.github/workflows/crowdin-upload.yml

```
@@ -0,0 +1,64 @@
+  workflow_dispatch:  # Allow manual triggering
+
+jobs:
+  extract-and-upload:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)